src/plugins/console/server/lib/spec_definitions/json/generated/indices.get_mapping.json

```diff
@@ -20,6 +20,6 @@
       "_mapping",
       "{indices}/_mapping"
     ],
-    "documentation": "https://opensearch.org/docs/latest/guide/en/elasticsearch/reference/master/indices-get-mapping.html"
+    "documentation": "https://opensearch.org/docs/latest/field-types/mappings/"
```

Comment:
I think it is better resource because get mapping is directly documented on this page. Here is the screenshot 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/67782303/228707754-b7dca1ca-7a53-4df9-a381-97865bf52523.png">

Reply:
I will update it to the link with anchor so that it will automatically scroll to the required part of the page.
https://opensearch.org/docs/latest/field-types/mappings/#get-a-mapping